pom.xml

```diff
@@ -156,6 +156,12 @@
         <!-- update-aem-deps:derived-from=org.osgi.util.function:1.2.0.202109301733 -->
         <version>1.2.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.osgi</groupId>
+        <artifactId>org.osgi.util.promise</artifactId>
+        <!-- update-aem-deps:derived-from=org.osgi.util.function:1.2.0.202109301733 -->
```

Comment:
@stefanseifert This is IMHO the wrong symbolic name, this is exposed/derived from `org.osgi.util.promise`

Reply:
oh, indeed - good catch. fixed in https://github.com/wcm-io/io.wcm.maven.aem-cloud-dependencies/commit/2bef21f8784130f25351eb1688fbfdbe889f827d.